charts/pomerium/templates/secret.yaml

```diff
@@ -53,10 +53,15 @@ stringData:
 {{- end -}}
 
 {{- end -}}
+{{- if .Values.config.insecure }}
+    {{- $_ := set .Values.config "scheme" "http" }}
+{{ else }}
+    {{- $_ := set .Values.config "scheme" "https" }}
+{{- end }}
 {{- if and .Values.forwardAuth.enabled .Values.forwardAuth.internal }}
-    forward_auth_url: https://{{ template "pomerium.proxy.fullname" . }}.{{ .Release.Namespace }}
+    forward_auth_url: {{ .Values.config.scheme }}://{{ template "pomerium.proxy.fullname" . }}.{{ .Release.Namespace }}
 {{ else }}
-    forward_auth_url: https://{{ template "pomerium.forwardAuth.name" . }}
+    forward_auth_url: {{ .Values.config.scheme }}://{{ template "pomerium.forwardAuth.name" . }}
```

Comment:
I think this should be `https` unless it is internal and insecure.  If the URL is being hit externally, it should be secure.